README.md

```diff
@@ -11,7 +11,7 @@ Requirements
 ------------
 
 -	[Terraform](https://www.terraform.io/downloads.html) 0.10.x
```

Comment:
Should this now be 0.11.x?

Reply:
We can leave it at 0.10.x. I've ran the acceptance tests with terraform 0.10.1 and it produced the same results as with 0.11. There should be no breaking changes between the terraform provider SDK versions.